server/src-lib/Hasura/RQL/Types/Helpers.hs

```diff
@@ -0,0 +1,8 @@
+module Hasura.RQL.Types.Helpers where
+
+import           Hasura.EncJSON
+
+successMsg :: EncJSON
+successMsg = "{\"message\":\"success\"}"
```

Comment:
Because `successMsg` seemed to be quite the odd one out in both those modules :P 

TBH, I wanted to put this in some `Utils` module instead of a `Types` module.